query-node/schemas/forum.graphql

```diff
@@ -0,0 +1,227 @@
+type ForumModerator @entity {
```

Comment:
I am not sure I see the need for this entity. A forum moderator is just a worker in the corresponding working group.